gather.go

```diff
@@ -225,7 +199,7 @@ func (a *Agent) gatherCandidatesLocal(ctx context.Context, networkTypes []Networ
 					Address:   address,
 					Port:      connAndPort.port,
 					Component: ComponentRTP,
-					TCPType:   tcpType,
+					TCPType:   connAndPort.tcpType,
```

Comment:
Can we rename this simply to `cfg`? And the corresponding `conns` to `cfgs`?

Reply:
I prefer more verbose naming. What about connectionConfiguration?